int32 num_1
int32 num_2
int32 num_3
int32 num_4
int32 num_5
int32 num_6
int32 num_7
int32 num_8
int32 num_9
---
float32 determinant
